server/controllers/chartsController.js

```diff
@@ -93,7 +97,7 @@ export const graphDataController = async (dataDb, userAccess, chart_id) => {
         : undefined
 
       if (studyTotals[study]) {
-        if (!!studyTotals[study].targetTotal) {
+        if (studyTotals[study].targetTotal !== undefined) {
```

Comment:
`undefined` means "we do not have a target total". Using an integer value coerced to a boolean could be misleading if the integer is `0`.